schemas/config/themes.json

```diff
@@ -0,0 +1,297 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "color": {
+      "type": "string",
+      "pattern": "^Transparent|(#[0-9A-Fa-f]{6})$"
+    },
+    "decimal": {
+      "type": "string",
+      "pattern": "^\\d+(\\.\\d+)?$"
+    }
+  },
+  "type": "object",
+  "patternProperties": {
+    "^[a-zA-Z_][a-zA-Z0-9_]*$": {
```

Comment:
@psyirius @ChrisTitusTech By selecting every theme entry, it'll make the tests always fail as now you don't need every single theming option in all theming entries, meaning the rule of having every theming option **required** should be set to the default theme.

more info can be found in [this PR](https://github.com/ChrisTitusTech/winutil/pull/2630)